### ISSUE TITLE
'ConfigProto' object has no attribute 'name'

### ISSUE THREAD

**Issue by powenlo** [tensorflow/tensorflow#21158]
Dear guys!!

I have encountered an issue when I train the neural network. It shows 'ConfigProto' object has no attribute 'name' in the terminal and I have no idea what's going on. Does anyone face similar problem and how to solve it. Thanks a lot.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 35 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
We are closing this issue for now due to lack of activity. Please comment if this is still an issue for you. Thanks!
